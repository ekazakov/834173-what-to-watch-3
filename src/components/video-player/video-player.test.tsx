import * as React from "react";
import * as renderer from "react-test-renderer";
import VideoPlayer from "./video-player";

const film: {
  src: string;
  poster: string;
} = {
  src: `https://upload.wikimedia.org/wikipedia/commons/transcoded/b/b3/Big_Buck_Bunny_Trailer_400p.ogv/Big_Buck_Bunny_Trailer_400p.ogv.360p.webm`,
  poster: `https://unsplash.it/280/175/`,
};

it(`VideoPlayer is renderer correctly`, () => {
  const tree = renderer.create(
      <VideoPlayer
        isPlaying={false}
        src={film.src}
        poster={film.poster}
      />, {
        createNodeMock: () => {
          return {};
        }
      }
  ).toJSON();

  expect(tree).toMatchSnapshot();
});
