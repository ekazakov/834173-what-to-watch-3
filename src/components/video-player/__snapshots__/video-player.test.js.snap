// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`VideoPlayer is renderer correctly 1`] = `
<video
  height="175"
  muted={true}
  width="280"
/>
`;
